WordPress/src/main/java/org/wordpress/android/ui/prefs/MultiSelectRecyclerViewAdapter.java

```diff
@@ -0,0 +1,97 @@
+package org.wordpress.android.ui.prefs;
+
+import android.content.Context;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.RecyclerView;
+import android.util.SparseBooleanArray;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+
+import java.util.List;
+
+public class MultiSelectRecyclerViewAdapter extends RecyclerView.Adapter<MultiSelectRecyclerViewAdapter.ItemHolder> {
+    private List<String> mItems;
+    private SparseBooleanArray mItemsSelected;
+    private int mSelectedColor;
+    private int mUnselectedColor;
+
+    public MultiSelectRecyclerViewAdapter(Context context, List<String> items) {
+        this.mSelectedColor = ContextCompat.getColor(context, R.color.white);
+        this.mUnselectedColor = ContextCompat.getColor(context, R.color.transparent);
+        this.mItems = items;
+        this.mItemsSelected = new SparseBooleanArray();
+    }
+
+    public class ItemHolder extends RecyclerView.ViewHolder {
+        private TextView text;
+        private View container;
+
+        public ItemHolder(View view) {
+            super(view);
+            this.text = (TextView) view.findViewById(R.id.text);
+            this.container = (LinearLayout) view.findViewById(R.id.container);
+        }
+    }
+
+    @Override
+    public int getItemCount() {
+        return mItems.size();
+    }
+
+    @Override
+    public void onBindViewHolder(final ItemHolder holder, int position) {
+        String item = mItems.get(holder.getAdapterPosition());
+        holder.text.setText(item);
+        holder.container.setBackgroundColor(
+                isItemSelected(position) ?
+                mSelectedColor :
+                mUnselectedColor
+        );
+    }
+
+    @Override
+    public ItemHolder onCreateViewHolder(ViewGroup parent, int type) {
+        return new ItemHolder(LayoutInflater.from(parent.getContext()).inflate(R.layout.wp_simple_list_item_1, parent, false));
+    }
+
+    public String getItem(int position) {
+        return mItems.get(position);
+    }
+
+    public SparseBooleanArray getItemsSelected() {
+        return mItemsSelected;
+    }
+
+    private boolean isItemSelected(int position) {
+        String item = getItem(position);
+        return item != null && mItemsSelected.get(position);
+    }
+
+    public void removeItemsSelected() {
+        mItemsSelected = new SparseBooleanArray();
```

Comment:
I think we can just call `clear()` on `mItemsSelected`, and by doing so we can also make it final.